notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>"This is useful when in conjugation with <code>add_data_on_build = True"</code></blockquote><p>I think "This can also be useful when used in conjunction with...". Or I would suggest just removing it all together. The add_data_on_build is surely a convenience, but just because it's set to false doesn't mean you can assume anything more about how many vectors are ultimately going to be added to the index. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
This one is a funny typo :)
Regarding the usefulness: I meant to say that if `add_data_on_build = False`, then `kmeans_trainset_fraction` doesn't make much sense: a user can just pass a smaller dataset to the effect.
I've added a small note to clarify this.